src/backend/libc/process/syscalls.rs

```diff
@@ -426,7 +426,7 @@ pub(crate) fn waitid(id: WaitId<'_>, options: WaitidOptions) -> io::Result<Optio
 #[cfg(not(any(target_os = "wasi", target_os = "redox", target_os = "openbsd")))]
 #[inline]
 fn _waitid_all(options: WaitidOptions) -> io::Result<Option<WaitidStatus>> {
-    let mut status = MaybeUninit::<c::siginfo_t>::uninit();
+    let mut status = MaybeUninit::<c::siginfo_t>::zeroed();
```

Comment:
Is the reason for explicit zeroing here the comment in the Linux man page about zeroing out the `si_pid` field? If so, could you quote that documentation here, so that it's recorded in the code why this uses zeroed memory?